nb/documentation/index.nb

```diff
@@ -3,9 +3,9 @@ md`
 
 An interactive overview of the basic features of JSxCAD.
 
-* [User Interface](../../nb/documentation/userinterface.md)
-* [Creating Geometry](../../nb/documentation/creatinggeometry.md)
-* [Interactions With Geometry](../../nb/documentation/interactionswithgeometry.md)
-* [Interactions Between Geometry](../../nb/documentation/interactionsbetweengeometry.md)
-* [File Import and Export](../../nb/documentation/fileimportandexport.md)
+* [User Interface](#JSxCAD@https://gitcdn.link/cdn/jsxcad/JSxCAD/${sha}/nb/documentation/user_interface.nb)
```

Comment:
How does the ${sha} work?

Reply:
It gets replaced with the commit to use -- by default it selects the latest one.